src/Microsoft.ML.Data/Transforms/TermTransform.cs

```diff
@@ -212,106 +377,148 @@ private CodecFactory CodecFactory
         /// <param name="maxNumTerms">Maximum number of terms to keep per column when auto-training.</param>
         /// <param name="sort">How items should be ordered when vectorized. By default, they will be in the order encountered.
         /// If by value items are sorted according to their default comparison, e.g., text sorting will be case sensitive (e.g., 'A' then 'Z' then 'a').</param>
-        public TermTransform(IHostEnvironment env,
+        public static IDataView Create(IHostEnvironment env,
             IDataView input,
             string name,
             string source = null,
             int maxNumTerms = Defaults.MaxNumTerms,
             SortOrder sort = Defaults.Sort)
-            : this(env, new Arguments() { Column = new[] { new Column() { Source = source ?? name, Name = name } }, MaxNumTerms = maxNumTerms, Sort = sort }, input)
         {
+            return Create(env, new Arguments() { Column = new[] { new Column() { Source = source ?? name, Name = name } }, MaxNumTerms = maxNumTerms, Sort = sort }, input);
```

Comment:
>new Arguments() [](start = 31, length = 15)

let's not use Args in this case

Reply:
also `=>`

---
In reply to: [214388347](https://github.com/dotnet/machinelearning/pull/759#discussion_r214388347) [](ancestors = 214388347)